.github/workflows/user-identity-mapper.yml

```diff
@@ -19,16 +19,16 @@ jobs:
 
```

Comment:
Can you remove GO_VERSION: from here too , it is not used anymore i guess

Reply:
whoops, good eye, I somehow missed this - thought i had removed it.
removed in 6950e0f